lib/src/components/motor/motor.dart

```diff
@@ -33,22 +33,71 @@ abstract class Motor extends Resource {
 
   Motor(this.name);
 
-  Future<void> setPower(double powerPct, {Map<String, dynamic>? extra});
```

Comment:
driveby: I changed this to from `powerPct` to `power` to match the python sdk / documentation. seemed minor enough to include this rename here.

This is the only code change in the PR.

Reply:
would keep at powerPct

Reply:
✅ back to `powerPct`